homeassistant/components/mastodon/__init__.py

```diff
@@ -17,14 +20,33 @@
 from homeassistant.helpers import discovery
 
 from .const import CONF_BASE_URL, DOMAIN
+from .coordinator import MastodonCoordinator
 from .utils import create_mastodon_client
 
+PLATFORMS: list[Platform] = [Platform.NOTIFY, Platform.SENSOR]
+
+
+@dataclass
+class MastodonData:
+    """Mastodon data type."""
+
+    client: Mastodon
+    instance: dict
+    account: dict
+    coordinator: MastodonCoordinator
+
+
+type MastodonConfigEntry = ConfigEntry[MastodonData]
+
+if TYPE_CHECKING:
+    from . import MastodonConfigEntry
```

Comment:
You don't need this one as it's defined here

Reply:
Oh whoops forgot that I made this comment when I approved it, could you please fix this one in a followup?

Reply:
Will do

Reply:
Fixed in https://github.com/home-assistant/core/pull/123467